policyengine_us/parameters/gov/states/ri/tax/income/deductions/standard/phase_out/increment.yaml

```diff
@@ -2,6 +2,7 @@ description: Rhode Island phases the standard deduction out at this increment.
 values:
   2021-01-01: 6_000
   2022-01-01: 6_200
+  2023-01-01: 6_700
```

Comment:
I think we need to uprate this as well:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/d132ff4a-489a-43e7-87b8-f380eb61ebed)